Description: Link examples with -lm, as some examples use functions like sin().
Author: Martin Pitt <martin.pitt@ubuntu.com>

--- a/inst-check-soxr
+++ b/inst-check-soxr
@@ -29,7 +29,7 @@
   for f in ?$2-*.[cC]; do
     cc=cc; echo $f | grep -q C$ && cc=c++
     out=$tmp/`echo $f | sed "s/.[cC]$//"`
-    cmd="$cc $cflags -o $out $f $libs"
+    cmd="$cc $cflags -o $out $f $libs -lm"
     echo $cmd; $cmd
   done
 }
